sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/persistence/impl/HibernateTissuePieceTypeDao.java

```diff
@@ -22,4 +23,11 @@ public long getUsage(TissuePieceType type) throws IOException {
     return getUsageBy(SampleTissuePieceImpl.class, "tissuePieceType", type);
   }
 
+  @Override
+  public TissuePieceType getByName(String name) {
+    return (TissuePieceType) currentSession().createCriteria(TissuePieceType.class)
+        .add(Restrictions.eq("name", name))
+        .uniqueResult();
```

Comment:
Can shorten to `return getBy("name", name);` (superclass method)

Reply:
Nice method!